build_targets

```diff
@@ -11,6 +11,7 @@ courbet user no no
 d1 userdebug yes yes
 d2s userdebug yes yes
 d2x userdebug yes yes
+f62 userdebug yes yes
```

Comment:
you may remove branch if default is 13.0
https://github.com/crdroidandroid/android_device_samsung_f62/blob/13.0/crdroid.dependencies#L5
and
https://github.com/crdroidandroid/android_device_samsung_f62/blob/13.0/crdroid.dependencies#L10

anyway, no show stopper